polyglot/piranha/test-resources/go/feature_flag/system_1/const_same_file/configurations/rules.toml

```diff
@@ -0,0 +1,61 @@
+# Copyright (c) 2022 Uber Technologies, Inc.
+#
+# <p>Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+# except in compliance with the License. You may obtain a copy of the License at
+# <p>http://www.apache.org/licenses/LICENSE-2.0
+#
+# <p>Unless required by applicable law or agreed to in writing, software distributed under the
+# License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+[[rules]]
+name = "false_flag"
+groups = ["replace_expression_with_boolean_literal"]
+query = """
+(
+    (call_expression
+        function: (selector_expression
+            operand: (_)
+            field: (field_identifier) @func_id
+        )
+        arguments: (argument_list
+            (interpreted_string_literal) @arg_str_literal
+        )
+    )
+    (#eq? @func_id "BoolValue")
+    (#eq? @arg_str_literal "\\\"@stale_flag_name\\\"")
+) @call_exp
+"""
+replace = "false"
+replace_node = "call_exp"
+holes = ["stale_flag_name"]
+
+[[rules]]
+name = "delete_const_str_literal"
+query = """
+(
+	(const_spec
+        name: (identifier) @const_id
+        value: (expression_list
+            (interpreted_string_literal) @const_str_literal
+        )
+    ) @const_spec
+   (#eq? @const_str_literal "\\"@stale_flag_name\\\"")
+)
+"""
+replace = ""
+replace_node = "const_spec"
+holes = ["stale_flag_name"]
+
+[[rules]]
+name = "replace_identifier_with_str_literal"
```

Comment:
what is the purpose of these rules?

Reply:
Thanks for the review, @ketkarameya. I agree that this rule is just noise. I restructured this test to have two rules:

- `find_const_str_literal`: finds a constant (`@const_id` with a value equal to `stale_flag_name` piranha_argument.
- `false_flag`: replaces with `false` calls to `BoolValue()` with an `identifier` argument equal to `@const_id` (hole).

Now the test doesn't remove the constant declaration.